mldsa/rounding.h

```diff
@@ -10,7 +10,31 @@
 #include "params.h"
 
 #define power2round MLD_NAMESPACE(power2round)
-int32_t power2round(int32_t *a0, int32_t a);
+/*************************************************
+ * Name:        power2round
+ *
+ * Description: For finite field element a, compute a0, a1 such that
+ *              a mod^+ MLDSA_Q = a1*2^MLDSA_D + a0 with -2^{MLDSA_D-1} < a0 <=
+ *              2^{MLDSA_D-1}. Assumes a to be standard representative.
+ *
+ * Arguments:   - int32_t a: input element
+ *              - int32_t *a0: pointer to output element a0
+ *              - int32_t *a1: pointer to output element a1
+ *
+ * Reference: a1 is passed as a return value instead
+ **************************************************/
+void power2round(int32_t *a0, int32_t *a1, int32_t a)
```

Comment:
Parameter "a" should be marked "const" both here and in the .c file.

Reply:
Isn't `const` meaningless for parameters passed by value? 

https://softwareengineering.stackexchange.com/questions/204500/when-and-for-what-purposes-should-the-const-keyword-be-used-in-c-for-variables
https://google.github.io/styleguide/cppguide.html#Use_of_const

suggest you should not use it for the function declarations as it provides no value to the caller. You can use it in the function definition, but here I am not sure if it adds any value as the function has just two lines that obviously do not modify `a`. 

I'd prefer to keep it as is.

Reply:
Agree. `const` for by-value parameters carries no information in declarations.

Reply:
I stand corrected! I never realized it can be "const" in the definition, but not in the declaration! I'd still prefer it to be const in the defintion, since it prevents the implementation modifying it, and provides an optimization hint I presume.

Reply:
Okay - I added it to rounding.c